tests/test_metadata_import.py

```diff
@@ -32,7 +32,10 @@ def test_no_start_time_fails(tmpdir, ref_data_dir):
     with pytest.raises(ValueError) as exc_info:
         with NwbFile(nwb_path, 'w') as nwb:
             nwb.create_from_metadata(meta_path, user="A")
-    assert "Start time for session not found!" == str(exc_info.value)
+    expected_error_string = "Start time for session not found! " \
+                            "Please add a `start_time` parameter to sessions for user A, " \
+                            "e.g. '10 May 2020 12:34:56.789103'"
+    assert expected_error_string == str(exc_info.value)
```

Comment:
As this gets longer, maybe it's better to check only the beginning e.g.
```python
assert str(exc_info.value).startswith(...)
```
or
```python
assert "Start time for session not found!" in str(exc_info.value)
```
or even add a `match=...` argument to `pytest.raises`.
We lose some detail with any of these, but that only matters if we care about the exact content of the error message. In this case, maybe we can also assert that it contains `start_time`, since that is the essence of the change.